### PR TITLE
Readme.

### PR DESCRIPTION
**Pull request by anne-gropler**
Please review the readme for readability, comprehensibility, completeness and accurancy (where appropriate).

### CODE REVIEW

**Review by Lichtso — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,9 +4,154 @@ Font Asset Generator based on OpenLL Specifications
...
+
+
+## Details on Algorithms and Parameters
+
+<dl>
+  <dt>Font Name</dt><dd>The font is loaded from the local machine using the given font name, e.g. Arial, Verdana</dd>
+  <dt>(Original) Font Size</dt><dd>This size is used to pre-render the glyphs before the Distance Transform is applied. A higher value results in smoother fonts (high resolution font), but the Distance Transform performs slower. We encourage larger font size, as the Distance Field is only generated once.</dd>
+  <dt>Padding</dt><dd>Space that is added around the glyphs. Gives more space for dynamic range of distance transformed glyphs. That means, the glyphs get 'larger' and thus need more space to not be cut off.</dd>
+  <dt>Downsampling</dt><dd>Distance Fields are meant to be downsampled. We offer different downsampling types and also different scaling factors.</dd>
+  <dt>Dynamic Range</dt><dd>The Distance Transform calculates values, that need to be clamped in order to generate a PNG. Choose the min and max values.</dd>
```
Comment (Blaidd-Drwg) [inline README.md]: Although this is the most objectively correct description, the effect of the dynamic range on the distance field is difficult to understand. In the help text of `llassetgen-cmd` I described it like this, maybe you could write something similar?

>Takes two values, BLACK and WHITE. A lower black value will make the distance fields wider; a lower white value will make the distance fields brighter. In most cases, the black value should be lower than the white value. However, swapping the black and white value will invert the colors of the atlas

Comment (anne-gropler) [reply README.md]: You are also assigned on this. Just correct it :) I like your description.

Comment (anne-gropler) [reply README.md]: Nevermind.

**Comment by anne-gropler** [comment]
@ all 
I'd like at least one more approvement before merging. @Blaidd-Drwg ?

**Review by Blaidd-Drwg — APPROVED** [review]
Sure, LGTM.
Sorry for ignoring your earlier comment btw, for some reason GitHub isn't sending me mails.

**Pull request merged**